src/one_dragon/base/controller/pc_screenshot/bitblt_screencapper.py

```
@@ -0,0 +1,194 @@
+        hwnd = self.game_win.get_hwnd()
+        if not hwnd:
+            return None
+        width = rect.width
+        height = rect.height
+        if width <= 0 or height <= 0:
+            return None
+        if independent:
+            return self._capture_independent(hwnd, width, height)
+        with self._lock:
+            if self.hwndDC is None or self.mfcDC is None:
+                if not self.init():
```

Ruff (0.12.2)

Use a single `if` statement instead of nested `if` statements

(SIM102)